### DISCUSSION TITLE
Globally set HStack to align="normal"

### DISCUSSION THREAD

**Discussion by jfrankl** [chakra-ui/chakra-ui#2201] [Q&A] [answered]
I am hoping to globally set the `HStack` component to `align="normal"`. Currently, I am adding that attribute to each instance, but I suspect there is a way to do this through the theme.

I added the following to my theme, but it doesn't seem to be working.

```
components: {
    HStack: {
        baseStyle: {
            align: 'normal',
        },
    },
}
```

Am I doing something wrong? Is there a better way to handle this? Thank you for taking a look.

**Comment by chasinhues** [comment, marked as answer]
Chakra theme doesn't seem to support modifying layout components like `HStack`, `Flex`, etc. from the theme.

If this is something you find yourself doing often, you could create a common component to contain the logic and use that instead.

```jsx
export const NormalAlignedHStack: React.FC = ({children}, props) => <HStack align="normal" {...props}>{children}</HStack>
```

**Reply by jfrankl** [reply]
Thanks for your reply, this is really helpful. Are you suggesting that there is a way to add this new component so that it could be imported with all of the other Chakra components (`import {HStack, NormalAlignedHStack, Box}`), or would this be a separate component imported separately.

**Reply by chasinhues** [reply]
Nah you would import this separately. That part is a bit of a pain but the benefit is the logic is contained for reusability and if you need to change the logic for all uses of the component you just do it in one spot 🙂
